packages/context/src/index.ts

```diff
@@ -7,6 +7,9 @@ import { FullViewingKey, WalletId } from '@penumbra-zone/protobuf/penumbra/core/
 import { ChainRegistryClient } from '@penumbra-labs/registry';
 import { AppParameters } from '@penumbra-zone/protobuf/penumbra/core/app/v1/app_pb';
 import { AssetId } from '@penumbra-zone/protobuf/penumbra/core/asset/v1/asset_pb';
+import Penumbra1Genesis from './penumbra-1-genesis.json';
+import { CompactBlock } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/compact_block/v1/compact_block_pb';
+import { JsonValue } from '@bufbuild/protobuf';
```

Comment:
Is there generally a distinction for using `JsonValue` over `JsonObject` to represent the genesis blob?

Reply:
I suppose it doesn't matter given a `JsonObject` is a type of `JsonValue`. That said, it's more accurate to make this a `JsonObject` I'd say.

Reply:
agreed, changed to `JsonObject`